### DISCUSSION TITLE
Advent Calendar idea cross post

### DISCUSSION THREAD

**Discussion by ktown007** [PerlDancer/Dancer2#1702] [Show and tell]
https://github.com/perladvent/Perl-Advent/issues/302

**Comment by cromedome** [comment]
I am really grateful for you putting this together. This year, however, we already had articles in the pipeline involving a todo app and one with htmx. That being said, I'd like to link to your Perl Advent Calendar article from the Dancer website. Does that work for you?

Anything we can do to support your article for the Perl Advent Calendar?

Thanks for helping to raise awareness of Dancer2!
-- Jason

**Comment by ktown007** [comment]
The current article uses an array for the data. The final version uses palnetscale db and fly.io deployment. You can see it here:
https://github.com/ktown007/todoapp
feel free to link both.

ken

**Comment by ktown007** [comment]
The Advent post is here: https://perladvent.org/2023/2023-12-05.html

**Reply by cromedome** [reply]
Thank you! I'll get it linked up to Dancer's website soon.
